public/css/calendar.less

```diff
@@ -338,6 +425,10 @@
     }
   }
 
+  .column-title:not(:first-of-type) {
+    border-left: 1px solid @gray-lighter;
```

Comment:
This doesn't align with the grid. Why is this needed?

Reply:
I would have to align this, but it is needed. Because in case of Week, it could be confusing to differentiate the recipients if you have many recipients in an entry.

Reply:
As discussed offline this change will be removed from this PR, and an [issue](https://github.com/Icinga/icinga-notifications-web/issues/168) has been opened for this.